src/main/java/uk/gov/hmcts/reform/civil/model/genapplication/GASuperClaimType.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.civil.model.genapplication;
```

Comment:
I do not understand the requirement of this class... where is it used?

Reply:
After talking to Smita - I deleted these values as they were not used and forgot to delete after changing the code. Tested again works approx.